harness/determined/_trial_controller.py

```diff
@@ -1,12 +1,17 @@
 import abc
 import logging
+import os
 from typing import Any, Optional, Type
-
+from distutils.util import strtobool
+from enum import Enum
 import determined as det
 from determined import horovod, profiler, workload
 from determined.common import check
 from determined.horovod import hvd
 
+class DistributedBackend(Enum):
```

Comment:
this needs a better place to live. I don't feel like it belongs in trial context or harness though